modules/tinymce/src/plugins/accordion/test/ts/browser/QuirksTest.ts

```diff
@@ -31,4 +32,14 @@ describe('browser.tinymce.plugins.accordion.QuirksTest', () => {
     editor.dispatch('click', event );
     TinyAssertions.assertCursor(editor, [ 1, 0 ], 0);
   });
+
+  it('TINY-11753: pressing enter when a figure is selected should move teh selection in the text into the figcaption', () => {
```

Comment:
Maybe alter it to this and fix the typo:

```suggestion
  it('TINY-11753: pressing enter when a figure is selected should move the selection into the figcaption', () => {
```

If we change it to other position types not just text then also add a case for a IMG element inside the caption and also add a custom one like a div with two spans or something where the first one is editable?

Reply:
changed